.gitignore

```diff
@@ -21,5 +21,6 @@ Thumbs.db
 .idea
 pubspec.lock
 .vscode/
+compile_commands.json
```

Comment:
Compile commands should only end up in `out/`. This should not be necessary.

Reply:
Done!